services/gemma-local/example_client.py

```
@@ -0,0 +1,197 @@
+    import soundfile as sf
+    
+    # Generate test audio
+    sample_rate = 16000
+    duration = 2.0  # 2 seconds
+    frequency = 440  # A4 note
+    
+    t = np.linspace(0, duration, int(sample_rate * duration))
+    audio_data = 0.3 * np.sin(2 * np.pi * frequency * t).astype(np.float32)
+    
+    # Save to temporary file
+    test_file = Path("/tmp/test_audio.wav")
```

Ruff (0.12.2)

Probable insecure usage of temporary file or directory: "/tmp/test_audio.wav"

(S108)